ideas/307-breaking-whiisper.md

```diff
@@ -0,0 +1,131 @@
+---
+id: 307-breaking-whisper
+title: Breaking Whisper
+status: Draft
+created: 2018-09-24
+category: swarm / SITG experiment
+lead-contributor: corpetty
+contributors:
+exit-criteria: no
+success-metrics: yes
+clear-roles: yes
+future-iterations: yes
+---
+
+## Preamble
+    Idea: 307-breaking-whisper
+    Title: SITG Experiment #2: Breaking Whisper
+    Status: Draft
+    Created: 2018-09-24
+
+## Summary
+1) Understanding the limits of whisper (with and without optimizations)
+
+## Swarm Participants
+- Lead Contributor: @corpetty 
+- Commited Contributors: 
+- Suggested Contributors (??): 
+  * Ivan: @divan
+  * Jakub: @jacob
+  * Jacek: @arnetheduck
+  * Mark: @easye
+  * Adam: @adamb
+
+## Overview
+Approach finding the limits of whisper from a rigorous academic approach.  See [this](https://hackmd.io/B-W4T1SzSgCAbYS2CfoIPg) for details.  We will be using the Skin In The Game (SITG) model for incentivizing the work here.  See [this](https://hackmd.io/eqS67u1QSfmZQ3IsgywFkQ) for more details on SITG.
+
+### Description
+The concept is simple: We use whisper as a fundamental backbone to Status' communication, but we don't know how much of a load that backbone can support.  Since this information does not exist, and there doesn't seem to be other people in the community focused on providing it, we must do it ourselves.
+
+This endeavor outlines the process of systematically and rigorously evaluating the limitations, strengths, and weaknesses of Whisper as a communications protocol.  It is then planned to use the results of this study to reason about future improvements of whisper and their prioritization, as well as what whisper's long term role within Status will be. 
+
+In an attempt to streamline and incentivize this process, as well as experiment with primitives of a DAO, we will be encapsulating this project with the SITG experiments, which are outlined in the above HackMD docs. 
+
+### Requirements & Dependencies
+- Whisper Simulator and metrics
+
+## [Milestones](#milestones)
+### Milestone 1: Simulation engine completion and metrics definition.
+Goal Date: 2018-
+Description: 
+* Completion of [whispervis](https://github.com/status-im/whispervis) and the required stats as laid out in Product Overview HackMD doc.
```

Comment:
Could we put the product overview inline here?

Reply:
yeah, I'll add it.